### PR TITLE
Ef/about page

### PR DESCRIPTION
**Pull request by coastlines**

### CODE REVIEW

File: src/portfolio-rebuild/about-page/about.html [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Erica Renée Faulkenberry</title>
+  <link href="https://fonts.googleapis.com/css?family=Poppins:900|Roboto+Mono&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="/reset.css">
+  <head>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/animate.css/3.7.2/animate.min.css">
+  </head>
```
Comment (juan-rod) [inline src/portfolio-rebuild/about-page/about.html:11]: multiple head tags?

Comment (juan-rod) [reply src/portfolio-rebuild/about-page/about.html:11]: don't think you need this

**Review by juan-rod — APPROVED** [review]
very nice work! merge when ready!

File: src/portfolio-rebuild/about-page/about.html (hunk 1 above)
Comment (coastlines) [reply src/portfolio-rebuild/about-page/about.html:11]: whoops yeah that is a mistake -- thanks!

**Pull request merged**